quickwit/quickwit-indexing/src/split_store/indexing_split_cache.rs

```diff
@@ -244,12 +318,13 @@ impl InnerLocalSplitStore {
     /// return true and record split , if the split should be moved into the cache.
     async fn make_room_and_record_split(&mut self, split_folder: SplitFolder) -> io::Result<bool> {
         // We don't accept splits that are too large.
-        if split_folder.num_bytes > self.split_store_space_quota.max_num_bytes() {
+        if split_folder.num_bytes > self.split_registry.quota().max_num_bytes() {
             return Ok(false);
         }
 
         while !self
-            .split_store_space_quota
+            .split_registry
+            .quota()
             .can_fit_split(split_folder.num_bytes)
         {
             self.evict_one_split().await?;
```

Comment:
do we prevent `max_num_splits = 0` and `max_num_bytes` isn't somewhere? If not, this can panic